src/main/target/common_pre.h

```diff
@@ -230,6 +230,11 @@
 #define USE_RPM_LIMIT
 #endif
 
+#ifdef USE_OSD
+// CMS is defined outside this block.
+#define USE_QUICK_OSD_MENU
+#endif
```

Comment:
```suggestion
#ifdef USE_OSD
// CMS is defined outside this block.
#define USE_QUICK_OSD_MENU
#define USE_SPEC_PREARM_SCREEN
#endif
```

Reply:
We can add this new macro as well (which does not yet exist). But if we wait for PR https://github.com/betaflight/betaflight/pull/13210 to be merged and then rebase this PR, we can kill two birds with one stone

Reply:
Changed my mind. This is bugfix. Not including in CI is another bug as we are adding another feature too. Rather like this to be separate. Have moved the defines to another PR https://github.com/betaflight/betaflight/pull/13212 which will need rebase after this one is merged. People already trying this feature so please merge asap.

Build fail
(FLYWOOF405S_AIO / 4.5.0-zulu) [182.232.216.154 / 10.10.0-debug-66bc189].
Log: https://build.betaflight.com/api/builds/025832b2ae1bc2b4ef5ab77df56019c3/log 
Request: https://build.betaflight.com/api/builds/025832b2ae1bc2b4ef5ab77df56019c3/json 
--//-- [18.7]

Build fail
(TMOTORF7 / 4.5.0-zulu) [24.119.194.83 / 10.10.0-debug-07da339].
Log: https://build.betaflight.com/api/builds/340b33b45cb70aabddc891821813c62a/log 
Request: https://build.betaflight.com/api/builds/340b33b45cb70aabddc891821813c62a/json 
--//-- [19.8]